### ISSUE TITLE
[EVM] Deploy on Celo

### ISSUE THREAD

**Issue by ccerv1** [hypercerts-org/hypercerts#682]
### Describe the feature you'd like to request

Celo is a highly mission aligned chain. It would be great to deploy on Celo when we have the remaining smart contract functions built out.

### Describe the solution you'd like

Hypercert contracts and graph support for Celo
Nice to have: dapp integration

### Describe alternatives you've considered

none

**Comment by daviddao** [comment]
+1 for this feature @ryscheng @holkeb 
we would love to leverage hypercerts for our communities at maps.gainforest.app

**Comment by bitbeckers** [comment]
Going to do this after #1082

**Comment by bitbeckers** [comment]
https://celoscan.io/address/0x16bA53B74c234C870c61EFC04cD418B8f2865959#code

**Comment by daviddao** [comment]
Yes! I'm so stoked for this!
